#! /usr/bin/env python
PACKAGE = "heron_nav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("goal_x",double_t,0,"The x position of goal",20.0,0,100)
gen.add("goal_y",double_t,0,"The y position of goal",20.0,0,100)
gen.add("goal_w",double_t,0,"The orientation of goal",1.0,0,100)

exit(gen.generate(PACKAGE,"heron_nav","navigation"))
